backend/twitterbot/tournesolbot.py

```diff
@@ -48,8 +49,11 @@ def prepare_tweet(video):
         CriteriaLocale.objects.filter(language=language).values_list("criteria__name", "label")
     )
 
-    # Replace "@" by "at" to avoid false mentions in the tweet
-    video_title = video.metadata["name"].replace("@", "at")
+    # Replace "@" by a smaller "@" to avoid false mentions in the tweet
+    video_title = video.metadata["name"].replace("@", "﹫")
+
+    # Replace "." in between words to avoid in the tweet false detection of links
+    video_title = re.sub(r"\b(?:\.)\b", "｡", video_title)
```

Comment:
The ["One Dot Leader" character](https://www.compart.com/en/unicode/U+2024) may be a more discreet alternative:
```suggestion
    video_title = re.sub(r"\b(?:\.)\b", "․", video_title)
```

Reply:
Nice! I missed this one